### ISSUE TITLE
Unable to write for longer than a few minutes after 3.2 upgrade from 3.1

### ISSUE THREAD

**Issue by dfedick** [scylladb/scylla#6989]
Report UUID:  1c9965bb-0062-4da9-bf60-27ecf183deba

This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [x] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*

**Old Scylla version (or git commit hash):** 3.1.1-0.20191024.f32ec885c

**New Scylla version:** 3.2.5-0.20200418.77500f9171

**Cluster size:** 9 nodes.

**OS (RHEL/CentOS/Ubuntu/AWS AMI)**: Ubuntu 16.04 

*Hardware details (for performance issues)*:  Instance Type:   i3en.3xlarge
Platform (physical/VM/cloud instance type/docker): EC2 Iinstances

*Hardware:*
API Name | Memory | vCPUs | Instance Storage | Network Performance
-- | -- | -- | -- | --
i3en.3xlarge | 96.0 GiB | 12 vCPUs | 7500 GiB NVMe SSD | Up to 25 Gigabit

**Comment by slivne** [comment]
@dfedick 
- I tried to download the info from 1c9965bb-0062-4da9-bf60-27ecf183deba and was not able to find anything - can you please double check that you uploaded to that directory 
- can you paste the upload command

**Comment by dfedick** [comment]
I just re-ran the following:
```
# curl -X PUT http://upload.scylladb.com/1c9965bb-0062-4da9-bf60-27ecf183deba/output_files.tgz -T ./output_files.tgz
```
and
```
curl -X PUT http://upload.scylladb.com/1c9965bb-0062-4da9-bf60-27ecf183deba/10.10.2.150-health-check-report.txt -T ./pzscylla-017ed8db84dd920c8.vdl.e2nova-health-check-report.txt
```

(I renamed the name of the file so you had context of which IP this server ran as in the logs.)

**Comment by dfedick** [comment]
working on gathering prometheus data and pushing.  taking a while to cobble together.

**Comment by dfedick** [comment]
I just added prom_data.tar.gz
```
# curl -X PUT http://upload.scylladb.com/1c9965bb-0062-4da9-bf60-27ecf183deba/prom_data.tar.gz -T ./prom_data.tar.gz
```

**Comment by dfedick** [comment]
Well, I'm not sure what the problem was.  But the fix for us was to keep upgrading to 4.1. Once we did that we turned on our ingest and everything looks great! :D
